### ISSUE TITLE
API for capturing pilot buttons

### ISSUE THREAD

**Issue by tverbele** [frankaemika/libfranka#7]
Is there any option (planned?) to capture button press events from the pilot interface?

**Comment by ch1bo** [comment]
Yeah we have planned a Pilot interface similar to the one available for the Hand end-effector. However, we currently focus on some more important bug fixes, improving stability and error handling. I will update this issue once we have implemented this feature, stay tuned!

**Comment by danneboom** [comment]
Is there any progress on that?

**Comment by ckfourie** [comment]
Seconding this. Would be very helpful.
